ding/worker/collector/sample_serial_collector.py

```diff
@@ -34,7 +34,8 @@ def __init__(
             policy: namedtuple = None,
             tb_logger: 'SummaryWriter' = None,  # noqa
             exp_name: Optional[str] = 'default_experiment',
-            instance_name: Optional[str] = 'collector'
+            instance_name: Optional[str] = 'collector',
+            timer_cuda: bool = False,
```

Comment:
add corresponding comments about the newly added argument

Reply:
fixed